pymeasure/instruments/hp/hp33120A.py

```diff
@@ -113,6 +123,67 @@ def __init__(self, adapter, name="Hewlett Packard 33120A Function Generator", **
         map_values=True
     )
 
+    burst_enabled = Instrument.control(
+        "BM:STATE?", "BM:STATE %d",
+        """Control state of burst modulation""",
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True
+    )
+
+    burst_source = Instrument.control(
+        "BM:SOURCE?", "BM:SOURCE %s",
+        """Control internal or external gate source for burst modulation""",
+        validator=strict_discrete_set,
+        values=['INT', 'EXT'],
+    )
+
+    burst_count = Instrument.control(
+        "BM:NCYC?", "BM:NCYC %d",
+        """Control the number of cycles per burst (1 to 50,000 cycles)""",
+    )
+
+    min_burst_count = Instrument.measurement(
+        "BM:NCYC? MIN",
+        """Get the minimum :attr:`~.HP33120A.burst_count`"""
+    )
+
+    max_burst_count = Instrument.measurement(
+        "BM:NCYC? MAX",
+        """Get the maximum :attr:`~.HP33120A.burst_count`"""
+    )
+
+    burst_rate = Instrument.control(
+        "BM:INT:RATE?", "BM:INT:RATE %g",
+        """Control the burst rate in Hz fo an internal burst source"""
+    )
+
+    min_burst_rate = Instrument.measurement(
+        "BM:INT:RATE? MIN",
+        """Get the minimum :attr:`~.HP33120A.burst_rate`"""
+    )
+
+    max_burst_rate = Instrument.measurement(
+        "BM:INT:RATE? MAX",
+        """Get the maximum :attr:`~.HP33120A.burst_rate`"""
+    )
+
+    burst_phase = Instrument.control(
+        "BM:PHAS?", "BM:PHAS %g",
+        """Control the starting phase angle of a burst (-360 to +360 degrees)"""
+    )
+
+    min_burst_phase = Instrument.measurement(
+        "BM:PHAS? MIN",
+        """Get the minimum :attr:`~.HP33120A.burst_phase`"""
+    )
+
+    max_burst_phase = Instrument.measurement(
+        "BM:PHAS? MAX",
+        """Get the maximum :attr:`~.HP33120A.burst_phase`"""
+    )
+
+
```

Comment:
Here you have too many blank lines: 2 instead of 1, which causes one check to fail.
```suggestion
```